definitions/tools/add_vep_fields_to_table.cwl

```diff
@@ -4,31 +4,30 @@ cwlVersion: v1.0
 class: CommandLineTool
 label: "add VEP annotation to report"
 
-baseCommand: ["/usr/bin/python", "/usr/bin/add_annotations_to_table_helper.py"]
+baseCommand: ["vep-annotation-reporter"]
 requirements:
     - class: DockerRequirement
-      dockerPull: "mgibio/annotation_table-cwl:1.0.3"
+      dockerPull: "mgibio/vcf_annotation_tools-cwl:3.0.0"
 arguments:
-    - position: 4
-      valueFrom: $(runtime.outdir)
+    ["-o", { valueFrom: $(runtime.outdir)/variants.annotated.tsv }]
 inputs:
-    tsv:
-        type: File
-        inputBinding:
-            position: 1
     vcf:
         type: File
         inputBinding:
-            position: 2
+            position: 1
     vep_fields:
         type:
             type: array
             items: string
         inputBinding:
             prefix: ""
-            itemSeparator: ","
+            itemSeparator: " "
             separate: false
```

Comment:
Not something you changed here, but the `prefix` and `separate` keys could be removed from this `inputBinding` entirely.